src/mount.ts

```diff
@@ -0,0 +1,48 @@
+import * as enzyme from 'enzyme';
+import type { ReactWrapper } from 'enzyme';
+import type { VNode } from 'preact';
+
+let containers: HTMLElement[] = [];
+let wrappers: ReactWrapper[] = [];
+
+export type MountOptions = {
+  /**
+   * If true, the element will be rendered in a container element which is
+   * attached to `document.body`.
+   */
+  connected?: boolean;
+};
```

Comment:
This is convenient, as very frequently we need the element to actually be in the DOM and we don't care about the container, but there are cases in which we need to customize the container with styles and such. See [this test](https://github.com/hypothesis/frontend-shared/blob/54d82eaaff583de6d7ecadd38feacbe6e61b360d/src/components/feedback/test/Popover-test.js#L51-L69) for example.

Maybe we could allow an HTMLElement to be conditionally provided, either by allowing the `connected` option to be `connected?: boolean | HTMLElement`, or by providing a more complex union type where either `connected?: boolean` or `container?: HTMLElement` to be allowed.

The later is probably more correct from a type point of view, but will require more checks.

We could also try to find a different option name where `boolean | HTMLElement` feels less awkward.

Reply:
Yeah, good point.

Let's land this. It will allow us to remove a lot of duplicated code already.